docs_requirements.txt

```diff
@@ -7,7 +7,9 @@ elasticsearch==7.10.1
 Jinja2==2.11.3
 Markdown==3.3.6
 MarkupSafe==2.0.1
+git+https://github.com/cfpb/mkDOCter.git@upgrade-for-mkdocs-1.2.3
 mkdocs==1.2.3
 mkDOCter==1.0.5
+>>>>>>> main
```

Comment:
🤨

Reply:
fixed

Reply:
I was adding instructions on how to test and was amid a cleanup.